src/manageiq_client/api.py

```diff
@@ -105,23 +105,44 @@ def _sending_request(self, func, retries=2):
                 retries -= 1
         raise last_connection_exception
 
-    def get(self, url, **get_params):
-        self.logger.info("[RESTAPI] GET %s %r", url, get_params)
+    def get(self, api_endpoint_url=None, **get_params):
+        if not api_endpoint_url:
+            if "url" not in get_params:
+                raise ValueError(
+                    "'api_endpoint_url' or 'url' key must be specified in the parameters."
```

Comment:
My only comment is consider not documenting a name at all, encourage passing as positional parameter.
(I reviewed various callers and all did positional anyway, except 1 place in integration_tests that I'll fix)
Something like:
```suggestion
                    "url must be specified, as positional paramater, or (deprecated) 'url' keyword parameter"
```
your call, LGTM either way (enough bikeshedding on my part ;-).

Reply:
I like your way, I was skeptical about the message but didn't know what needed fix, this looks good.